waspc/src/Wasp/Analyzer/Parser/Lexer.x

```diff
@@ -79,13 +79,11 @@ startCodeToInt (QuoterStartCode _) = quoter
 --
 --   This function is taken from the Alex basic wrapper.
 alexGetByte :: AlexInput -> Maybe (Word8, AlexInput)
-alexGetByte (prevChar, (currChar, (b:bs)), remainingSource) =
-  Just (b, (prevChar, (currChar, bs), remainingSource))
-alexGetByte (_, (_, []), []) = Nothing
-alexGetByte (_, (currChar, []), (newChar:remainingSource)) =
-  case encodeChar newChar of
-    (b:bs) -> Just (b, (currChar, (newChar, bs), remainingSource))
-    [] -> Nothing
+alexGetByte (prevChar, (b:bs), remainingSource) = Just (b, (prevChar, bs, remainingSource))
```

Comment:
Couple of PRs ago I added this tracking of current char but it turned out we don't need it and it just complicates things so I removed it again.